test/live-reloading/python/mpg-estimator/cortex.yaml

```diff
@@ -5,4 +5,5 @@
   predictor:
     type: python
     path: predictor.py
-    model_path: s3://cortex-examples/sklearn/mpg-estimator/linreg/
+    dynamic_model_loading:
+      model_path: s3://cortex-examples/sklearn/mpg-estimator/linreg/
```

Comment:
This needs to be changed to an iris-classifier example that has at least 3 versions.